### DISCUSSION TITLE
[Connector-requst] Birdeye

### DISCUSSION THREAD

**Discussion by aazam-gh** [airbytehq/airbyte#47408] [New Connector Request]
Topic

This is a suggestion to create a connector for Birdeye using the UI Connector Builder. The website for this is https://birdeye.com/
Relevant information

**Comment by aazam-gh** [comment]
Please assign me this

**Reply by jdbranham** [reply]
Nice @aazam-gh !

Any news on this? 
Also saw you commented on this one - https://github.com/airbytehq/airbyte/discussions/47268

[a comment by an automated account is not shown]
